demo/angularjs/components/expansion-panel/doc.md

```diff
@@ -0,0 +1,23 @@
+# Expansion panel
+
+**Expansion panels deliver large amounts of content in a small space through <a href="https://en.wikipedia.org/wiki/Progressive_disclosure" target="_blank">progressive disclosure</a>.**
+
+## Default
+
+<demo-block component="expansion-panel" partial="default"></demo-block>
+
+## Drag option
+
+A drag handle can be added.
+
+<demo-block component="expansion-panel" partial="drag"></demo-block>
+
+## Trimmed
+
+Use trimmed when you don't need the left and right spacing.
```

Comment:
if `trimmed`is a named option it must be capitalized
Do not use contraction.
I do not understand this sentence -> which spacing? the margin? the padding?